packages/expo-google-sign-in/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "expo-google-sign-in",
-  "version": "1.0.0",
+  "version": "1.1.0",
```

Comment:
Yeah, bumping the version number is a part of publishing modules script, so there is no need to bump it here if we're not going to publish it before the release, but it will work as expected as the script is smart enough 😄 In other words, if version `1.1.0` is higher than the last published version (`1.0.0` in that case) then the script will use this version as a default version to publish.